1. Desktop app calls Web API/README-incremental.md

```diff
@@ -337,7 +337,7 @@ See [2. Web API now calls Microsoft Graph](../2.%20Web%20API%20now%20calls%20Mic
 
 Use [Stack Overflow](http://stackoverflow.com/questions/tagged/msal) to get support from the community.
 Ask your questions on Stack Overflow first and browse existing issues to see if someone has asked your question before.
-Make sure that your questions or comments are tagged with [`azure-active-directory` `msal` `dotnet`].
+Make sure that your questions or comments are tagged with [`microsoft-entra-id` `msal` `dotnet`].
```

Comment:
Was the "azure-active-directory" tag removed in Stack overflow. It seems to me that it enables people to troubleshoot themselves. I would keep it here (find historical issues?)

Reply:
Great catch. I have added this entry to the post-transform section of the script to correct this issue.` @{ Key = '`microsoft-entra-id`'; Value = '`azure-active-directory`' }`

I will run the updated script on this branch to address this issue everywhere and hopefully will prevent this issue in all the other repos that will be updated.

Reply:
thank you!

Reply:
This readme file has been updated with the correct tag now. Thank you again.